ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestconnectivity.py

```diff
@@ -163,6 +163,54 @@ def test_definition(self) -> dict | None:
         return {self.anta_module: anta_tests} if anta_tests else None
 
 
+class AvdTestDpsReachability(AvdTestBase):
+    """
+    AvdTestDpsReachability class for DPS reachability tests.
+    """
+
+    anta_module = "anta.tests.connectivity"
+
+    @cached_property
+    def test_definition(self) -> dict | None:
+        """
+        Generates the proper ANTA test definition for all DPS reachability tests.
+        Returns:
+            dict | None: ANTA test definition if there are tests to run, otherwise None.
+        """
+
+        anta_tests = []
+
+        # Skip the test if the host is not a VTEP (no VXLAN interface)
+        if not self.is_vtep():
+            LOGGER.info("Host is not a VTEP since it doesn't have a VXLAN interface. %s is skipped.", self.__class__.__name__)
+            return None
+
+        if not self.is_wan_vtep():
+            LOGGER.info("Host is not a WAN VTEP. %s is skipped.", self.__class__.__name__)
+            return None
```

Comment:
this could be only one check is `is_wan_vtep` was dependent on `is_vtep`

Reply:
done